REVIEW.md

```diff
@@ -0,0 +1,192 @@
+It is a great way to get familiar with the codebase and should motivate the contributor to keep involved in the project.
+
+- Every PR is an act of generosity.
+  Opening with a positive comment will help the author feel rewarded, and your subsequent remarks may be heard more clearly.
+  You may feel good also.
+- Begin if possible with the large issues, so the author knows they’ve been understood.
+Resist the temptation to immediately go line by line, or to open with small pervasive issues.
+- Do not let perfect be the enemy of the good.
+
+If you find yourself making many small suggestions that don’t fall into the Review Guidelines, consider the following approaches:
+    - Refrain from submitting these.
+    - Follow up in a subsequent PR, out of courtesy, you may want to let the original contributor know.
+    - Do not rush, take the time to make your comments clear and justify your suggestions in a small number of words (ideally a small paragraph or a short sentence should be enough)
+    - You are the face of the project.
+
+Bad days occur to everyone, in that occasion you deserve a break: try to take your time and stay offline.
+
+### Expected reviewer behavior
+
+Pull requests are a key space for community interaction, and reviewing PRs is a vital part of maintaining any project.
+With that in mind, this is a process that can benefit from having a clear set of goals for the sake of both the reviewer and the PR author.
+
+All reviewers are expected to follow these guidelines and labels in their review.
+If a reviewer forgets, they may be reminded.
+If a reviewer refuses, they will be contacted about a violation of [Spyder’s Code of Conduct](https://github.com/spyder-ide/spyder/blob/master/CODE_OF_CONDUCT.md) under `Giving and gracefully accepting constructive feedback`.
+As with all aspects of the Code of Conduct, this applies to everyone involved in the discussion (including members of the Spyder organization on GitHub).
+
+Other notes:
+- Unless the reviewer has explicit permission on the conversation for that PR, reviewers should not commit to the same branch as the PR being reviewed.
+- If more than roughly 20% of review comments fall in the `[optional]` section described below, then it is worth adding this feedback in an issue linked to the PR instead of on the PR directly.
+- When possible, please add comments asking for a specific change as [GitHub suggestions](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request).
+
+## Reviewing process
+
+### For code-focused review
+
+#### Required review
+
+The following are areas of review the community expects to be covered in the review of every PR.
+Not all of the following questions will be relevant for every PR (for example, a bug fix may not need to answer the same questions as a proposal for a new feature), but these are the themes to be addressed.
```

Comment:
```suggestion
Not all of the following individual questions will be relevant for every PR (for example, a bug fix may not need to answer the same questions as a proposal for a new feature), but these are the themes to be addressed.
```

Distinguish this a bit more clearly from the above (otherwise, on the surface they might read a little contradictory)